cases/management/commands/enrich_ciaa_allegations.py

```
@@ -54,553 +179,822 @@
+                ctx.minimum_version = ssl.TLSVersion.TLSv1_2
+                resp = urllib.request.urlopen(req, timeout=timeout, context=ctx)
+                payload = json.loads(resp.read().decode("utf-8"))
+                raw = _parse_llm_opencode_response(payload, is_minimax)
+                logger.debug(f"LLM response: {raw[:500]}...")
+                return raw
+
+            except urllib.error.HTTPError as e:
+                last_status = e.code
+                try:
+                    last_body_snippet = e.read().decode("utf-8", errors="replace")[:500]
+                except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)